### PR TITLE
FIPS version

### PR DESCRIPTION
**Pull request by danvangeest**
- NIST OIDs
- Private Key Encodings
- More generally PKIX instead of certificates + CRLs
- More like RFC 8410 than dilithum-certificates
- Examples

### CODE REVIEW

**Review by BashiriK — APPROVED** [review]

File: draft-ietf-lamps-x509-slhdsa.md [hunk 1]
```diff
@@ -152,80 +148,80 @@ The fields in AlgorithmIdentifier have the following meanings:
...
-    CertificateList  ::=  SEQUENCE  {
-        tbsCertificate       TBSCertList,
-        signatureAlgorithm   AlgorithmIdentifier,
-        signatureValue       BIT STRING  }
-~~~
-
-The identifiers defined in {{sec-alg-ids}} can be used as the AlgorithmIdentifier in the signatureAlgorithm field in the sequence Certificate/CertificateList and the signature field in the sequence TBSCertificate/TBSCertList in certificates CRLs, respectively, {{!RFC5280}}.  The parameters of these signature algorithms are absent, as explained in {{sec-alg-ids}}.
-
-The signatureValue field contains the corresponding SLH-DSA signature computed upon the ASN.1 DER encoded tbsCertificate {{!RFC5280}}.
+The same algorithm identifiers in {{sec-alg-ids}} are used for signatures as are used
```
Comment (fluppe2) [inline draft-ietf-lamps-x509-slhdsa.md:202]: Is there a doubling of "are used" in "are used for signatures as are used for public keys"?

File: draft-ietf-lamps-x509-slhdsa.md [hunk 2]
```diff
@@ -372,6 +365,53 @@ If the keyUsage extension is present in a certificate that indicates an id-alg-s
...
+NOTE: There exist some private key import functions that have not
+picked up the new ASN.1 structure OneAsymmetricKey that is defined in
+{{RFC5958}}.  This means that they will not accept a private key
+structure that contains the public key field.  This means a balancing
+act needs to be done between being able to do a consistency check on
+the key pair and widest ability to import the key.
+
 # Security Considerations
 
 The security considerations of {{RFC5280}} applies accordingly.
```
Comment (fluppe2) [inline draft-ietf-lamps-x509-slhdsa.md:417]:
```suggestion
The security considerations of {{RFC5280}} apply accordingly.
```

**Review by fluppe2 — CHANGES_REQUESTED** [review]
LGTM

Just some minor editorial comments that should be checked prior to merging.

File: draft-ietf-lamps-x509-slhdsa.md (hunk 1 above)
Comment (danvangeest) [reply draft-ietf-lamps-x509-slhdsa.md:202]: That is fine in English, I think it's the "in {{sec-alg-ids}}" that makes it awkward and isn't necessary so I'll remove that.

**Pull request merged**